LEAF_Nexus/js/dialogController.js

```diff
@@ -18,10 +18,15 @@ function dialogController(containerID, contentID, indicatorID, btnSaveID, btnCan
 	this.validatorOks = new Object();
 	this.invalid = 0;
 
+	//calculate min width of dialog based on min width of content div
+	var minWidth = parseInt($('#' + this.contentID).css('min-width'));
+	minWidth = (minWidth == 0) ? 0 : (minWidth + 30);
+
 	$('#' + this.containerID).dialog({autoOpen: false,
 										modal: true,
 										height: 'auto',
-										width: 'auto'});
+										width: 'auto',
```

Comment:
Fix indentation

Reply:
The indentation is fine in the file itself locally.

Reply:
Please replace tab spacing with 4 spaces